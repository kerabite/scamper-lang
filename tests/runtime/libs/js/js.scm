(import js)

(define example
  (js-obj "foo" 0 "bar" 1 "baz" 2))

example

(js-get example "foo")
(js-get example "bar")
(js-get example "baz")

(define str "  hello world!     ")

(js-method str "trim")

(js-eval "1+1")