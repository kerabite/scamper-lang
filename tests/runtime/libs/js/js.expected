[object Object]
0
1
2
"hello world!"
2
